src/client/Timesheet.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+
+interface TableParameter {
+    dateValue: string;
+    hoursValue: number;
+    rateValue: number;
+    amountValue: number;
+}
+
+interface Props {
+    date: string;
+    hours: number;
+    rate: number;
+    amount: number;
+}
```

Comment:
Hint: this interface needs only one property inside.

PS Please check out comment above.